SDK v2 migration guide.md

```diff
@@ -16,182 +16,189 @@ are outlines of the notable breaking changes as well as a mapping from version 1
+  > We feel this caused more harm than good and as such it was best to remove.
 - Some options that were previously set in the `DeviceClient` constructor are now in the optional `IotHubClientOptions` parameter.
 - The connection status callback parameters have changed. Instead of two parameters, a class with several properties is provided.
-  - Two properties are the same as before, but with some renames (underscores removed) and obsolete members removed.
+  - Two properties enum values are mostly the same as before, with some renames (underscores removed) and obsolete members removed.
   - A new property has been added with a recommended action, which a device developer may observe or ignore.
 - The file upload method has been split into the three individual steps that this method used to take. See [this file upload sample](./iothub/device/samples/getting%20started/FileUploadSample/) for an example of how to do file upload using these discrete steps.
 - Cloud-to-device messages can be received by calling `SetMessageHandlerAsync` and providing a callback. Users no longer need to poll for messages with `ReceiveAsync`.
-- Several callback handler set methods and definitions have changed, losing the `userContext` parameter.
+- Several callback handler set methods and definitions have changed, losing the `userContext` parameter which was deemed unnecessary and a vestige of the C device client.
 - The exponential back-off retry policy has updated parameters and logic.
 - Remote certificate validation is no natively longer supported for AMQP web socket connections.
   - The supported workaround is to provide a client web socket instance in the client options.
 - The authentication classes for devices and modules have been consolidated.
+- Reduced access levels to classes and methods that were never intended to be public where possible.
 
 #### Notable additions
 
 - The device and module clients now have a property (e.g., `IotHubDeviceClient.ConnectionStatusInfo`) with the latest connection status information on it, eliminating the need for a connection status callback method to cache the latest values.
 - Added support for setting a client web socket instance in the client options so that users can have better control over AMQP web socket connections.
-- The library now includes IncrementalDelayRetryStrategy and FixedDelayRetryStrategy.
-- The client can now be re-opened after it has been closed. It cannot be re-opened after it has been disposed, though. Also, subscriptions do not carry over when the client is re-opened.
+- The library now includes 2 new `IRetryPolicy` implmentations: `IncrementalDelayRetryStrategy` and `FixedDelayRetryStrategy`.
+- The client can now be re-opened after it has been closed, removing the need to close/dispose/initialize/open; simply initialize/open, then if necessary close/open, and finally when done dispose.
+  > Be advised, subscriptions do not carry over when the client is re-opened.
+  >
```

Comment:
extra line?

Reply:
Yes, it ensures the notes aren't all on the same line. A simple newline won't separate them.